packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -1021,10 +1023,11 @@ class _MenuItemButtonState extends State<MenuItemButton> {
     // Since we don't want to use the theme style or default style from the
     // TextButton, we merge the styles, merging them in the right order when
     // each type of style exists. Each "*StyleOf" function is only called once.
-    final ButtonStyle mergedStyle =
-        widget.style?.merge(widget.themeStyleOf(context)?.merge(widget.defaultStyleOf(context))) ??
-            widget.themeStyleOf(context)?.merge(widget.defaultStyleOf(context)) ??
-            widget.defaultStyleOf(context);
+    ButtonStyle mergedStyle = widget.themeStyleOf(context)?.merge(widget.defaultStyleOf(context))
```

Comment:
How is this any different from the previous version?  I think it evaluates to the same thing, right?

Reply:
This change is to handle the case when we don't have a custom "themeStyleOf". Previously, if the `widget.style` is not null, but the `widget.themeStyleOf(context)` is null, the first line of code `widget.style?.merge(widget.themeStyleOf(context)?.merge(widget.defaultStyleOf(context)))` will just return the `widget.style` without merging with the default theme data. So the button will only have the same style as the default TextButton:) Please let me know if I misunderstood anything.

Reply:
Ahh, OK, that makes sense.